services/azure-devops/azure-devops-build.service.js

```diff
@@ -46,7 +53,7 @@ module.exports = class AzureDevOpsBuild extends BaseSvgScrapingService {
         pattern: ':organization/:projectId/:definitionId',
         namedParams: {
           organization: 'totodem',
-          projectId: '8cf3ec0e-d0c2-4fcd-8206-ad204f254a96',
+          projectId: 'Shields.io',
```

Comment:
Could you please explain this change? I'm not familiar with how this service works.

Reply:
Good point: I noticed that Azure DevOps was now accepting the project name for Builds whereas it used to expect a project id. But I guess we should probably stick to the project id, especially because on Releases, Azure DevOps still require a project id. I'll rollback that part.